### ISSUE TITLE
Style Loader Invalid Options

### ISSUE THREAD

**Issue by paulbrie** [zeit/next-plugins#20]
Hi, I'd like to use TypeScript with `next-css`. For now I get this error:

![image](https://user-images.githubusercontent.com/3997538/35965420-77cebfa4-0cc3-11e8-95a8-f4a05ba6ac93.png)

This is my `next.config.js` file.
```javascript
const withCSS = require('@zeit/next-css')

module.exports = withCSS({
  cssModules: true,
  webpack: (config, options) => {
    const { dir, defaultLoaders } = options
    // add TypeScript support
    config.resolve.extensions.push('.ts', '.tsx')
    config.module.rules.push({
      test: /\.+(ts|tsx)$/,
      include: [dir],
      exclude: /node_modules/,
      use: [
        defaultLoaders.babel,
        { loader: 'ts-loader', options: { transpileOnly: true } }
      ]
    })

    return config
  }
})
```
I get the same with `module.exports = withCSS(withTypescript())`

Am I doing something wrong?

Thanks,
Paul

**Comment by fadiquader** [comment]
Without typescript!,  and get same Error!

**Comment by egorovli** [comment]
+1, same here:

```bash
 ERROR  Failed to compile with 1 errors                                                                 20:05:16
 error  in ./styles/main.scss

Module build failed: ValidationError: Style Loader Invalid Options
options['importLoaders'] should NOT have additional properties

 @ ./pages/index.jsx 19:0-29
 @ multi ./pages/index.jsx
```

`./styles/main.scss`:

```scss
html,
body {
  background-color: papayawhip;
}
```

`./pages/index.jsx`:

```jsx
import React, { Component, Fragment } from 'react'
import Link from 'next/link'
import '../styles/main.scss'

class Landing extends Component {
  render () {
    return (
      <Fragment>
        <p>hi there</p>
        <Link href='/test'>come here</Link>
      </Fragment>
    )
  }
}

export default Landing
```

**Comment by irfanabliz** [comment]
I am having the same error, not even working when I follow all the instruction step by step, it gaves me same error message as above for sass and scss

**Comment by ataquino** [comment]
Try it like this:

```
module.exports = withCSS({
  cssModules: true,
  webpack (config, options) {
    const { dir, defaultLoaders } = options
    // add TypeScript support
    config.resolve.extensions.push('.ts', '.tsx')
    config.module.rules.push({
      test: /\.+(ts|tsx)$/,
      include: [dir],
      exclude: /node_modules/,
      use: [
        defaultLoaders.babel,
        { loader: 'ts-loader', options: { transpileOnly: true } }
      ]
    })

    return config
  }
})
```

**Comment by fadiquader** [comment]
I solved by installing the dependencies below

  "dependencies": {
    "css-loader": "^0.28.9",
    "extract-text-webpack-plugin": "^3.0.2",
    "find-up": "^2.1.0",
    "ignore-loader": "^0.1.2",
    "postcss-loader": "^2.0.10",
    "style-loader": "^0.19.1"
  },
  "devDependencies": {
    "webpack": "^3.10.0"
  }

**Comment by httpete-ire** [comment]
same issue here

```
Module build failed: ValidationError: Style Loader Invalid Options

options['importLoaders'] should NOT have additional properties


 @ ./pages/components/alert.js 10:0-37
 @ multi ./pages/components/alert.js
```

---
`./pages/index.scss`:

```
body {
  background: red;
}
```
---

`./pages/index.jsx`:

```
import styles from './index.scss';

export default () => (
  <div>Hello world</div>
);
```

---

`next.config.js`:

```
const withSass = require('@zeit/next-sass');
module.exports = withSass({
  webpack(config, options) {
    config.module.rules.push({
      test: /\.svg$/,
      issuer: [/\.css$/, /\.(scss|sass)$/],
      loader: 'file-loader',
      query: {
        name: 'assets/icons/[name].[ext]',
      },
    });

    return config;
  },
});
```

**Comment by timneutkens** [comment]
Fixed in v0.1.0

**Comment by dryleaf** [comment]
Along with the `css` configurations mentioned here, What if I want to export also the following settings in `next.config.js`:
```
module.exports = {
  useFileSystemPublicRoutes: false,
  poweredByHeader: false
}
```
How can this be achieved?
